app/models/onboarding.rb

```diff
@@ -7,7 +7,6 @@ class Onboarding
   validates :programmes, presence: true
   validates :teams, presence: true
   validates :schools, presence: true
-  validates :clinics, presence: true
```

Comment:
I don't think we should make this change, one of the reasons the onboarding process was added to make it impossible to set up an organisation without clinics.

Without clinics, it's impossible to vaccinate patients in the "Community clinic" session as the nurse will be asked for which location and [there won't be any options to choose from](https://trello.com/c/XK3ye26w/1418-clinic-vaccinations-cannot-be-recorded-if-the-sais-org-is-missing-clinic-locations).

To prepare for the training tomorrow can we create a fake clinic for now?

Reply:
Yeah, I'll update with a fake clinic then 👍